packages/carbon-graphs/tests/unit/controls/Gantt/GanttPanning-spec.js

```diff
@@ -267,88 +267,88 @@ describe('Panning', () => {
       );
       expect(actionsContent.length).toEqual(2);
     });
-    it("EventlineGroup translates properly when panning is enabled", (done) => {
-      if(gantt) {
+    it('EventlineGroup translates properly when panning is enabled', (done) => {
+      if (gantt) {
         gantt.destroy();
       }
       const axisObj = utils.deepClone(getAxes(axisJSON));
       axisObj.eventline = utils.deepClone(eventlineJSON);
       gantt = new Gantt(axisObj);
       const eventlineGroup = fetchElementByClass(styles.eventlineGroup);
       delay(() => {
-        const translate = getSVGAnimatedTransformList(
-          eventlineGroup.getAttribute("transform")
-        ).translate;
+        const { translate } = getSVGAnimatedTransformList(
+          eventlineGroup.getAttribute('transform'),
```

Comment:
is this trailing comma needed?  Seems odd for an argument to a function call

Reply:
Yes according to linter rules Andy.